examples/rnn_classifer/model.py

```diff
@@ -0,0 +1,108 @@
+import os
+from urllib.request import urlretrieve
+import tarfile
+
+from dstoolbox.transformers import Padder2d
+from dstoolbox.transformers import TextFeaturizer
+import numpy as np
+from sklearn.datasets import load_files
```

Comment:
I followed the instructions, assuming that if I pasted all the code to model.py as instructed, it would work, but there are some parts missing. For instance, in the docs, this line is missing, leading to an import error.